integration-cli/docker_api_containers_test.go

```diff
@@ -1308,31 +1143,19 @@ func (s *DockerSuite) TestContainersApiChunkedEncoding(c *check.C) {
 	req.ContentLength = -1
 
 	resp, err := client.Do(req)
-	if err != nil {
-		c.Fatalf("error starting container with chunked encoding: %v", err)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("error starting container with chunked encoding"))
 	resp.Body.Close()
-	if resp.StatusCode != 204 {
-		c.Fatalf("expected status code 204, got %d", resp.StatusCode)
-	}
+	c.Assert(resp.StatusCode, checker.Equals, 204)
 
 	out, err = inspectFieldJSON(id, "HostConfig.Binds")
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, checker.IsNil)
 
 	var binds []string
-	if err := json.NewDecoder(strings.NewReader(out)).Decode(&binds); err != nil {
-		c.Fatal(err)
-	}
-	if len(binds) != 1 {
-		c.Fatalf("got unexpected binds: %v", binds)
-	}
+	c.Assert(json.NewDecoder(strings.NewReader(out)).Decode(&binds), checker.IsNil)
+	c.Assert(binds, checker.HasLen, 1, "Got unexpected binds: %v", binds)
```

Comment:
Looks like the format of this `Assert` needs to end with `check.Commentf`; otherwise the `HasLen` thinks it is getting extra parameters; for example should look like:
```
c.Assert(events, checker.HasLen, 3, check.Commentf("Events == %s", events))
```